.github/workflows/build.yml

```diff
@@ -0,0 +1,40 @@
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+      types: [opened, synchronize, reopened]
+name: Main Workflow
+jobs:
+  sonarcloud:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0 # Disable shallow clone to improve relevancy
+    - name: SonarCloud Scan
+      uses: sonarsource/sonarcloud-github-action@v1.5
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+  shellcheck:
+    name: ShellCheck
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Run ShellCheck
+      uses: ludeeus/action-shellcheck@master
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install bats and shellmock
+        run: |
+          sudo add-apt-repository ppa:duggan/bats
```

Comment:
Review from Zoom: Is there another repository available? Is this always up-to-date with [the GitHub source code](https://github.com/bats-core/bats-core)?

Reply:
Though [documented in the official bats-core wiki](https://github.com/bats-core/bats-core/wiki/Install-Bats-Using-a-Package#-ubuntu), it looks like bats is aware that [this will not necessarily get the latest version of bats](https://github.com/bats-core/bats-core/issues/103)